.husky/pre-commit

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
```

Comment:
does this work for you? I'm getting

<img width="453" alt="image" src="https://user-images.githubusercontent.com/1404810/203923838-e99cc046-b8bc-4a87-9f9b-94a558e0fa6a.png">

when I try to commit

Reply:
`npm bin` fails for me

<img width="480" alt="image" src="https://user-images.githubusercontent.com/1404810/203924037-726c1642-0ce6-4eca-a902-09a8253282e0.png">

using `npx` works - I can send a PR

Reply:
#457

Reply:
whaaat. What kind of NPM are you running? `npm bin` works even if `alias npm=pnpm`.

Reply:
I need to test this locally.

Reply:
![image](https://user-images.githubusercontent.com/618076/203944220-00a57c9b-760f-4630-8374-a9cbf97f35e5.png)

Works fine? Is not `npx` potentially just a lot slower?